src/controllers/project.js

```diff
@@ -1,9 +1,113 @@
 const { ObjectId } = require('mongodb');
 const elasticsearch = require('elasticsearch');
+const mongoose = require('mongoose');
 
 const Project = require('../models/project');
 const User = require('../models/user');
 
+const createProject = async (req, res) => {
+  try {
+    const { /* comment, review, //BUG not used */ project, user } = req.body;
+
+    const newProjects = new Project({
+      title: project.title,
+      description: project.description,
+      category: [],
```

Comment:
it should be 'categories' not category please check the project model to get the correct props